bin/upward.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
Not sure this belongs in the `bin` directory anymore, but `src` doesn't seem right either since it's a sample implementation. Going to throw out `dev/LocalPHPRouter.php`, but open to ideas.

Reply:
The more I think on it, the less confident _either_ of those files belongs in `bin`. I mean, `bootstrap-controller.php` is (primarily) a bootstrapper.

Reply:
`dev/bootstrap-controller.php` and `dev/router.php`?